pages/reference-react-hooks/createconfig.mdx

```diff
@@ -0,0 +1,85 @@
+import { Tabs } from "nextra/components";
+
+# `createConfig`
+
+```typescript
+createConfig<
+  Provider extends SdkStarterKitConfig["provider"] = SdkStarterKitConfig["provider"],
+  Signer extends SdkStarterKitConfig["signer"] = SdkStarterKitConfig["signer"],
+  Chain extends ChainType = ChainType
+>(params: CreateConfigParams<Provider, Signer, Chain>): SafeConfig<Provider, Signer, Chain>
+
+CreateConfigParams<
+  Provider extends SdkStarterKitConfig['provider'] = SdkStarterKitConfig['provider'],
+  Signer extends SdkStarterKitConfig['signer'] = SdkStarterKitConfig['signer'],
+  Chain extends ChainType = ChainType
+> = { chain: Chain; provider: Provider; signer: Signer } & SdkStarterKitConfig
+```
+
+The props for creating a new configuration are those extracted from [SdkStarterKitConfig](https://github.com/safe-global/safe-core-sdk/blob/main/packages/sdk-starter-kit/src/types.ts#L49-L65) and are extended with the viem chain.
+
+## Usage
+
+We can have primarily two use cases:
+
+- When using the `SafeProvider` component to provide a common configuration for all hooks.  
+- When you are not using the `SafeProvider` component or when using individual functions with a custom configuration.
```

Comment:
It's actually always required to use a `SafeProvider`. The parameter for the individual functions can only be used to define a config that should be used **instead** of the one from the nearest SafeProvider. But using a hook without being wrapped by a SafeProvider would still not work because the hooks rely on the [QueryClientProvider](https://tanstack.com/query/v4/docs/framework/react/reference/QueryClientProvider) and [WagmiProvider](https://wagmi.sh/react/api/WagmiProvider) which are [defined in the SafeProvider](https://github.com/safe-global/safe-core-sdk-react-hooks/blob/main/src/SafeProvider.ts#L81-L83).

Reply:
aaaah , didn't know

Reply:
fixing